feature-libs/customer-ticketing/components/customer-ticketing-list/customer-ticketing-list.component.ts

```diff
@@ -1,11 +1,161 @@
 import { Component, OnInit } from '@angular/core';
+import { of, combineLatest, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { TranslationService } from '@spartacus/core';
+import { STATUS, TEXT_COLOR_CLASS } from '@spartacus/customer-ticketing/root';
 
 @Component({
   selector: 'cx-customer-ticketing-list',
   templateUrl: './customer-ticketing-list.component.html',
 })
 export class CustomerTicketingListComponent implements OnInit {
-  constructor() {}
+  constructor(protected translation: TranslationService) {}
+  sortType: string = 'byId';
+  iconTypes = ICON_TYPE;
+  customerTicketsFlag: boolean = true;
 
-  ngOnInit() {}
+  tickets$ = of({
+    pagination: {
+      currentPage: 0,
+      pageSize: 5,
+      sort: 'byId',
+      totalPages: 2,
+      totalResults: 10,
+    },
+    sorts: [
+      { code: 'byId', selected: true },
+      { code: 'byChangedDate', selected: false },
+    ],
+    tickets: [
+      {
+        availableStatusTransitions: [
+          {
+            id: 'CLOSED',
+            name: 'Closed',
+          },
+        ],
+        id: '0000001',
+        createdAt: '2021-01-13T10:06:57+0000',
+        modifiedAt: '2021-01-13T10:06:57+0000',
+        status: {
+          id: 'CLOSED',
+          name: 'Closed',
+        },
+        subject: 'My drill is broken.',
+        ticketCategory: {
+          id: 'ENQUIRY',
+          name: 'Enquiry',
+        },
+        ticketEvents: [
+          {
+            author: 'Mark Rivers',
+            createdAt: '2021-01-13T10:06:57+0000',
+            message:
+              'It is broken when I receive it. Please send one replacement to me.',
+            toStatus: {
+              id: 'CLOSED',
+              name: 'Closed',
+            },
+          },
+        ],
+      },
+      {
+        availableStatusTransitions: [
+          {
+            id: 'CLOSED',
+            name: 'Closed',
+          },
+        ],
+        id: '0000002',
+        createdAt: '2021-01-14T10:06:57+0000',
+        modifiedAt: '2021-01-14T10:06:57+0000',
+        status: {
+          id: 'OPEN',
+          name: 'Open',
+        },
+        subject: 'Need fix for my door',
+        ticketCategory: {
+          id: 'ENQUIRY',
+          name: 'Enquiry',
+        },
+        ticketEvents: [
+          {
+            author: 'Bob',
+            createdAt: '2021-01-14T10:06:57+0000',
+            message: 'Door received broken',
+            toStatus: {
+              id: 'OPEN',
+              name: 'Open',
+            },
+          },
+        ],
+      },
+    ],
+  });
+
+  tickets = this.tickets$.pipe(
+    map((tickets) => {
```

Comment:
How about changing the pipe and subscribe returning variable to "result" or "ticketList" in order to avoid "tickets.tickets..."